src/test/java/SetTest.java

```diff
@@ -0,0 +1,55 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.*;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("setSize test")
+    void test1() throws Exception {
+        //given
+        //when
+        int expected = 3;
+        int result = numbers.size();
+        //then
+        assertThat(result).isEqualTo(expected);
```

Comment:
```suggestion
        int expected = 3;
        int result = numbers.size();

        assertThat(result).isEqualTo(expected);
```

`given, when, then` 구조화를 하면 테스트 코드 작성의 일관성, 구조화 되어 가독성 향상, 간결함 의 장점이 있을거라고 생각되는데요.

그렇다면, 테스트 코드를 간결하게 구현하는게 우선이 아닐까? 라는 생각이 들더라구요.  
그런의미에서 공백으로 구분 처리하면 어떨까요?  
지금처럼 `//given` 이 없는경우는 굉장이 애매하다고 생각하지 않으셨나요? 🥲

Reply:
```suggestion
        int expected = 3;
        int actual = numbers.size();

        assertThat(actual).isEqualTo(expected);
```

`expected`좋네요 👍 
`actual` 이런건 어떨까요? 검증하고자 하는 값의 변수에 대한 이름을 걱정할것 없이
테스트 코드에서의 의미를 일관되게 부여하는것이죠 😄

Reply:
사용하지 않는 given 공백 처리와 실제 값의 변수 이름 설정.. 감사합니다 잘 외우겠습니다..!